Makefile

```diff
@@ -22,6 +22,7 @@ DOCKER_REGISTRY_REPO    += mattermost
 DOCKER_USER             ?= user
 DOCKER_PASSWORD         ?= password
 ## Docker Images
+DOCKER_IMAGE_GO         += "golang:${GO_VERSION}@sha256:79138c839452a2a9d767f0bba601bd5f63af4a1d8bb645bf6141bff8f4f33bb8"
```

Comment:
Will same sha work for different go versions ?

Reply:
It will not. 
The user will have to update the SHA hash, if the go version defined under `go.mod` file changes

I think it is good tradeoff between automating the go version extraction, and at the same time, pining the image using the SHA.

Reply:
Thank you very much, it is clear now. Yes we need that tradeoff at the moment.